pkg/kf/commands/services/create-service.go

```diff
@@ -49,19 +59,103 @@ func NewCreateServiceCommand(p *config.KfParams, client services.ClientInterface
 			if err != nil {
 				return err
 			}
+			paramBytes, err := json.Marshal(params)
+			if err != nil {
+				return err
+			}
+			rawParams := &runtime.RawExtension{
+				Raw: paramBytes,
+			}
 
-			instance, err := client.CreateService(
-				instanceName,
-				serviceName,
-				planName,
-				services.WithCreateServiceNamespace(p.Namespace),
-				services.WithCreateServiceParams(params))
+			clusterPlans, err := client.ServicecatalogV1beta1().
+				ClusterServicePlans().
+				List(metav1.ListOptions{})
 			if err != nil {
 				return err
 			}
 
-			describe.ServiceInstance(cmd.OutOrStdout(), instance)
-			return nil
+			for _, plan := range clusterPlans.Items {
+				if planName != plan.Spec.ExternalName {
+					continue
+				}
+				if serviceName != plan.Spec.ClusterServiceClassRef.Name {
+					continue
+				}
+				if broker != "" && broker != plan.Spec.ClusterServiceBrokerName {
+					continue
+				}
```

Comment:
I think this logic should be broken into `findMatchingClusterPlans` and `findMatchingNamespacePlans` that both return every matching plan and if the total number of plans found is > 1 return an error

testing should also be easier if you're guaranteed to always call both